### DISCUSSION TITLE
❌ Sessions being timed out are irritating.

### DISCUSSION THREAD

**Discussion by ahiemstra** [isontheline/pro.webssh.net#959] [Ideas]
Whenever I look away for a minute, my session times out and I have to re-login. 

This is VERY irritating.

I have set the timeout and idle settings to 60 seconds, but this is not enough. 

I would like to see 15 minutes or so.

Regards, Arjen.

**Comment by isontheline** [comment]
Thanks for your feedback @ahiemstra 🙏

This is an iOS limitation not a WebSSH one.

So yet I don't know how to resolve this behavior 😞

**Reply by macjl** [reply]
Some other SSH app use location to maintain the application connected in the background. It seems to work, because I have sessions that stays connected for days in ont of this app.

Maybe you can also use this workaround ?

**Reply by isontheline** [reply]
No I should not : privacy and energy saving matters more.

I want a true Apple solution instead
